tests/do-your-own-map/pass_frenchris.js

```diff
@@ -0,0 +1,7 @@
+const map = (el, fn) => {
+  let result = []
+  for (let i in el) result.push(fn(el[i], i, el))
+  return result
+}
+
+const flatMap = (el, fn) => map(el, fn).flat()
```

Comment:
Should we authorize `flat` ?
Don't that make the exercise too easy ?

Reply:
Fair point, I restricted [].flat() and updated my solution accordingly